controllers/imageupdateautomation_controller.go

```diff
@@ -78,22 +80,41 @@ type ImageUpdateAutomationReconciler struct {
 func (r *ImageUpdateAutomationReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	ctx := context.Background()
 	log := r.Log.WithValues("imageupdateautomation", req.NamespacedName)
+	now := time.Now()
 
 	var auto imagev1.ImageUpdateAutomation
 	if err := r.Get(ctx, req.NamespacedName, &auto); err != nil {
 		return ctrl.Result{}, client.IgnoreNotFound(err)
 	}
 
+	auto.Status.LastAutomationRunTime = &metav1.Time{Time: now}
```

Comment:
This would update on every reconcile, regardless of success or failure, is that the intention?

Reply:
To be honest I did not think hard about exactly when this should be updated. Thinking about what is the purpose of the field (to inform the user), it should be updated when the automation run has completed without error, whether or not it made changes. I'll change it.

Reply:
Move it to the end, just before it sets the ready condition and updates the status for the last time.